### DISCUSSION TITLE
student development pack

### DISCUSSION THREAD

**Discussion by junaidshaik2600** [community/community#59313] [GitHub Education] [answered]
### Hi everyone,

i have been approved yesterday mens (25-06-2023) in student development pack but i still no get benifits ofo it why i still did not get ?

**Comment by jge162** [comment]
The time for it to become available is around 3-5 days. So you have still sometime to go.

**Reply by junaidshaik2600** [reply]
Thank bro

**Reply by jge162** [reply]
No worries

**Comment by samus-aran** [comment, marked as answer]
Hey @jge162  👋

When you submit your application, you'll have your academic affiliation verified and gain access to Global Campus right away if successful. Your educational benefits will arrive within 72 hours of this verification. However, during busy periods like the start of a new semester, it may take a bit longer.

You’ll get an email notification once your academic benefits are ready, or you can check your status [here](https://education.github.com/discount_requests/pack_application). Meanwhile, as a verified [GitHub Global Campus](https://education.github.com/globalcampus/student) student, you can explore the platform, connect with a local [Campus Expert](https://education.github.com/students/experts), attend events hosted by student leaders and GitHub Education, discover and contribute to student-created repositories from [GitHub Community Exchange](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/about-github-community-exchange) and much more!

 I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open and we'll be happy to assist!

**Reply by junaidshaik2600** [reply]
I got student development pack by the thanks.
